underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java

```diff
@@ -156,16 +156,16 @@ public boolean exists(String path) throws IOException {
   }
 
   /**
-   * Gets the block size in bytes. There is no concept of a block in OSS, however the maximum
-   * allowed size of one file is currently 5 GB.
+   * Gets the block size in bytes. There is no concept of a block in OSS and the maximum size of
```

Comment:
I think the comment here is not precise. According OSS docs: "In OSS, each file of the user is an object. The maximum size of a file varies with the method of data uploading. The maximum size of the object through the Put operation cannot exceed 5 GB, and the maximum size of the object through Multipart uploading cannot exceed 48.8 TB."

Similarly, for S3 "Individual Amazon S3 objects can range in size from 1 byte to 5 terabytes. The largest object that can be uploaded in a single PUT is 5 gigabytes. For objects larger than 100 megabytes, customers should consider using the Multipart Upload capability."